# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/ActorsChild.h
# Commit: 9e7a90dcc429
# Full Hash: 9e7a90dcc429194fbe08e63aeed311ce100114cc
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-10-02 03:38:52
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Use const where easily possible. r=ttung,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43631
# ==============================================================================

diff -r e493a6b5f125 -r 9e7a90dcc429 dom/indexedDB/ActorsChild.h
--- a/dom/indexedDB/ActorsChild.h	Tue Oct 01 08:56:37 2019 +0000
+++ b/dom/indexedDB/ActorsChild.h	Tue Oct 01 09:14:04 2019 +0000
@@ -210,7 +210,7 @@
   friend class PermissionRequestChild;
   friend class PermissionRequestParent;
 
-  RefPtr<IDBFactory> mFactory;
+  const RefPtr<IDBFactory> mFactory;
 
   // Normally when opening of a database is successful, we receive a database
   // actor in request response, so we can use it to call ReleaseDOMObject()